src/common/transformations/include/transformations/common_optimizations/fused_names_cleanup.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+#include <transformations_visibility.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API FusedNamesCleanup;
```

Comment:
I'm not sure that it should be a new transformation actually, why not just function?
it's internal stuff in my opinion

Reply:
> I'm not sure that it should be a new transformation actually, why not just a function? it's internal stuff in my opinion to remove these Fusing names

Because we want to propose plugins use this transformations at the begging of their transformation pipeline in QN if they needed it